### DISCUSSION TITLE
Any plans to address CVE-2022-37434?

### DISCUSSION THREAD

**Discussion by Collisio-Adolebitque** [keycloak/keycloak#13814] [Q&A]
Lead to believe that KeyCloak is vulnerable to a critical buffer overflow: https://nvd.nist.gov/vuln/detail/CVE-2022-37434.  Can anyone confirm and advise what the resolution is?

**Comment by abstractj** [comment]
@Collisio-Adolebitque could you please elaborate more? Which version of Keycloak do you believe is affected by the CVE-2022-37434 above? What makes you believe that Keycloak is vulnerable by the CVE mentioned?

**Reply by Collisio-Adolebitque** [reply]
Because Trivy flagged it in a scan.

**Comment by abstractj** [comment]
@Collisio-Adolebitque we don't have any Keycloak container image tagged as `16.1.1-debian-10-r11`. For official Keycloak images, please refer to:

- https://quay.io/repository/keycloak/keycloak?tab=tags

Added to that, it is strongly recommended to upgrade to the latest release (19.0.1). I hope it brings some clarity.

**Reply by Collisio-Adolebitque** [reply]
It was Keycloak version 16.1.1.

**Comment by abstractj** [comment]
@Collisio-Adolebitque that was certainly was addressed on the latest releases. Considering that we don't have enough bandwidth to maintain old images, please upgrade.
